docs/src/ref_impl/cross_compilation.md

```diff
@@ -22,19 +22,25 @@ To enable ``binfmt``, we recommend to set the following in your host systems ``c
       "riscv64-linux"
       "aarch64-linux"
     ];
-    
+
 In addition, it is recommended to enable KVM support with either
 
     boot.kernelModules = [ "kvm-amd" ];
 
 or
 
     boot.kernelModules = [ "kvm-intel" ];
-    
+
 depending on whether your development host is running AMD or Intel processor.
 
+## Cross Compilation for Microchip Icicle Kit (RISCV64)
+SD Image for Microchip Icicle kit can be cross compiled from x86 machine. To generate the release/debug SD Image run following command:
```

Comment:
Replace with:

An SD image for the Microchip Icicle Kit can be cross-compiled from an x86 machine. To generate the release or debug an SD image run the following command:

Reply:
Done.